tests/functional/syntax/exceptions/test_invalid_literal_exception.py

```diff
@@ -18,13 +18,6 @@ def foo():
     """,
     """
 @external
-def foo(x: int128):
-    y: int128 = 7
-    for i in range(x, x + y):
-        pass
-    """,
-    """
-@external
 def foo():
```

Comment:
does this raise a StructureException now?

Reply:
yeah this test was duplicated in other files

Reply:
hmm i did not actually see this one in any other files, can you point me to it?

Reply:
There's a lot of tests resulting in "Value must be a literal integer" in [test_for_range]. It already fails on validating the `x` and `x+y` isn't even looked at.
It's not exactly the same code, if you wish we can add this exact code to the tests

Reply:
Sorry here is the link
https://github.com/DanielSchiavini/vyper/blob/d161895/tests/functional/syntax/test_for_range.py#L79-L124

Reply:
either way, extra test can't hurt
https://github.com/vyperlang/vyper/pull/3679/commits/548ce58d1d875784310be0039b3784ecde70a8ad